src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-scanCredentialsId.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    <p>Credentials used to scan branches and check out sources</p>
+    <p>Credentials used to <strong>scan</strong> branches and pull requests, <strong>check</strong> out sources and <strong>mark</strong> commit statuses</p>
```

Comment:
The separable verb is `check out`, so it should either all be emphasized or none.